src/components/application_manager/test/request_controller/request_controller_test.cc

```diff
@@ -258,11 +241,23 @@ TEST_F(RequestControllerTestClass, IsLowVoltage_SetOnWakeUp_FALSE) {
 }
 
 TEST_F(RequestControllerTestClass, AddMobileRequest_SetValidData_SUCCESS) {
```

Comment:
This unit test seems to fail sometimes
```
      Start 17: request_controller_test
17/36 Test #17: request_controller_test .............Child aborted***Exception:   1.16 sec
97% tests passed, 1 tests failed out of 36

Total Test time (real) =  39.75 sec

The following tests FAILED:
	 17 - request_controller_test (Child aborted)
```

Seeing this error message in the Testing/Temporary/LastTest.log

```
[ RUN      ] RequestControllerTestClass.AddMobileRequest_SetValidData_SUCCESS
/home/shobhit/sdl/CORE/test/sdl_core/src/components/application_manager/test/request_controller/request_controller_test.cc:254: Failure
Actual function call count doesn't match EXPECT_CALL(app_mngr_, IsStopping())...
         Expected: to be called once
           Actual: never called - unsatisfied and active
pure virtual method called
terminate called without an active exception
<end of output>
Test time =   1.16 sec
----------------------------------------------------------
Test Failed.
```

Reply:
Hey @RomanReznichenkoLuxoft, the test seems to fail every 1/50 runs. No special conditions, I saw the test fail the first time I ran `make test` and just re-ran `make test` repeatedly until I encountered the issue again.

Reply:
Hi @ShobhitAd. Thanks, I also detected locally. Now I analyze the tests and life of objects. I will let you know when there is a fix.